src/lib/ggshield-results-parser.ts

```diff
@@ -45,6 +45,10 @@ function filterUriOccurrences(occurrences: Occurrence[]): Occurrence[] {
  * @param results ggshield scan results
  * @returns incidents diagnostics
  */
+
```

Comment:
- You probably don't need to use Number() here (num is already supposed to be a number)
- What about moving this into `src/utils`?
- I think you're breaking the JSDoc for `parseGGShieldResults` with placing the function here